### PR TITLE
fix OpenBLAS in Pangea3 host-config, add pygeosx to install process

### PR DESCRIPTION
**Pull request by XL64**

### CODE REVIEW

**Review by sframba — APPROVED** [review]
Seems to work, thanks

File: host-configs/TOTAL/pangea3-gcc8.4.1-openmpi-4.1.2.cmake [hunk 1]
```diff
@@ -92,23 +92,11 @@ set(ENABLE_ESSL OFF CACHE BOOL "")
 
 set(ENABLE_PYGEOSX ON CACHE BOOL "")
 
-if( ENABLE_ESSL )
```
Comment (rrsettgast) [inline host-configs/TOTAL/pangea3-gcc8.4.1-openmpi-4.1.2.cmake:95]: @XL64 what happened to ESSL on Pangea3.

**Comment by TotoGaz** [comment]
@XL64 Is this ready to be merged? (it should but I prefer to ask)

**Comment by XL64** [comment]
> @XL64 Is this ready to be merged? (it should but I prefer to ask)

It is ready indeed.

**Pull request merged**